DittoHeartbeat/src/main/java/live.ditto.dittoheartbeat/Heartbeat.kt

```diff
@@ -0,0 +1,149 @@
+package live.ditto.dittoheartbeat
+
+import live.ditto.Ditto
+import android.os.Handler
+import android.os.Looper
+import androidx.compose.material3.Text
+import live.ditto.DittoConnectionType
+import live.ditto.DittoPeer
+import live.ditto.DittoPresenceObserver
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+data class HeartbeatConfig(
+    val interval: Long,
+    val collectionName: String,
+    val metaData: Map<String, Any>
+)
+
+data class HeartbeatInfo(
+    val id: Map<String, String>,
+    val lastUpdated: String,
+    val presence: Presence?,
+)
+
+data class Presence(
```

Comment:
Based on how this class is used, seems like `totalConnections` is a derived property from `connections`
Could you also consider rename "connections" to "peers"?

```
data class Presence(
   val peers: List<DittoPeer>,
) 
```